src/Components/Wrapper/Wrapper.tsx

```diff
@@ -86,6 +89,22 @@ const Wrapper = (props: PropsWithChildren<{}>) => {
   const { configLoading, configResponse: config } = useGetConfig();
   const { language_options: languageOptions = {} } = config ?? {};
   const languages = Object.keys(languageOptions) as Language[];
+  const {
+    referrerData: referrerData = {
+      theme: {
+        default: 'default',
+        '211co': 'twoOneOne',
+      },
+      logoSource: coLogoSource,
+      logoAlt: coLogoAlt,
+      logoClass: {
+        default: 'logo',
+      },
+      twoOneOneLink: coTwoOneOneLink,
+      shareLink: coShareLink,
+    },
+  } = config ?? {};
```

Comment:
Don't worry about providing default values. If the config is undefined then just set the `referrerData` to undefined as well.
```suggestion
  const {
    referrerData = undefined
  } = config ?? {};
```